app/src/main/java/org/liberty/android/fantastischmemo/dao/CardDaoImpl.java

```diff
@@ -431,6 +431,32 @@ public long getScheduledCardCount(Category filterCategory, Date startDate, Date
         }
     }
 
+    public long getNewLearnedCardCount(Category filterCategory, Date startDate, Date endDate) {
+        try {
+            LearningDataDao learningDataDao = getHelper().getLearningDataDao();
+            QueryBuilder<Card, Integer> cardQb = queryBuilder();
+            QueryBuilder<LearningData, Integer> learnQb = learningDataDao.queryBuilder();
+            cardQb.setCountOf(true);
+            cardQb.selectColumns("id");
+            learnQb.selectColumns("id");
+
+            learnQb.where().le("firstLearnDate", endDate)
+                    .and().ge("firstLearnDate", startDate)
+                    .and().gt("acqReps", "0").prepare();
+
+            Where<Card, Integer> where = cardQb.where().in("learningData_id", learnQb);
+            if (filterCategory != null) {
+                where.and().eq("category_id", filterCategory.getId());
+            }
+            cardQb.setWhere(where);
+
+            return countOf(cardQb.prepare());
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
The stack trace printing was there because I wanted to be consistent with other methods. Most of the other methods in class has the stack trace printing. I just want to make sure that if you only want to have this stack trace printing removed, or if I should remove them for all methods in the class.